pom.xml

```diff
@@ -137,9 +142,24 @@
                 <artifactId>jib-maven-plugin</artifactId>
                 <version>${jib-maven-plugin.version}</version>
                 <configuration>
-                    <container>
-                        <expandClasspathDependencies>true</expandClasspathDependencies>
-                    </container>
+                  <container>
+                      <expandClasspathDependencies>true</expandClasspathDependencies>
+                  </container>
+                   <from>
+                          <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-21:latest</image>
+                   </from>
+                   <to>
+                            <!-- update once image moved to shared services -->
+                            <image>169942020521.dkr.ecr.eu-west-2.amazonaws.com/local/disqualified-officers-delta-consumer:latest</image>
```

Comment:
just quick change to replace disqualified-officers with bankrupt-officer-search-api :)